pallets/anmol-utils/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+authors = ['DotMatrix <hello@dotmatrix.im>']
+description = 'Anmol utils pallet'
+edition = '2018'
+homepage = 'https://anmol.network/'
+license = 'Unlicense'
```

Comment:
@justinFrevert @aelesbao Please keep the same license as the main repo all around